cfme/tests/cloud_infra_common/test_tag_objects.py

```diff
@@ -147,6 +170,24 @@ def test_tag_infra_objects(tagging_check, infra_test_item, tag_place):
     tagging_check(infra_test_item, tag_place)
 
 
+@pytest.mark.meta(automates=[1726313])
+@pytest.mark.provider([InfraProvider], selector=ONE)
```

Comment:
**Suggestion:** 
@lina-nikiforova `ONE_PER_CATEGORY` would be more useful than `ONE`, as BZ doesn't comprises of any specific provider.

Reply:
exactly, it doesn't depend on the provider, provider is needed only to have some entity to tag, therefore it doesn't make sense to run this tests against multiple providers

Reply:
Okay :+1: